doc/tired_support.md

```diff
@@ -0,0 +1,197 @@
+# Tired Supported
+
+**Tier definitions**
+
+- **A — Production Ready:** fully tested and stable.
+- **B — Almost Production Ready:** partially tested; close to production.
+- **C — Experimental / Not Production Ready:** unfinished or volatile.
+
+## Architecture Support
+
+| Architecture | Tier  |
+| ------------ | ----- |
+| **x86-64**   | **A** |
+| **x86-32**   | **A** |
+| AArch64      | B     |
+| ARC          | B[^1] |
+| ARM          | B     |
+| RISCV32      | B     |
+| RISCV64      | B     |
+| THUMB        | B     |
+| MIPS         | C     |
+| XTENSA       | C     |
+
+[^1]: will upgrade to **A** after further testing and validation.
+
+## OS / Platform Support
+
+| Platform           | Tier  |
+| ------------------ | ----- |
+| **NuttX**          | **A** |
+| **Ubuntu**         | **A** |
+| Android            | B     |
+| macOS              | B     |
```

Comment:
macos lower than nuttx? it doesn't match my impression. what's your criteria?

Reply:
Because there are products on Nuttx, and we haven't conducted a full test in CI on macOS like we have on Ubuntu.